engines/leveldb/leveldb_test.go

```diff
@@ -0,0 +1,235 @@
+package leveldb_test
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/rotationalio/honu/config"
+	engine "github.com/rotationalio/honu/engines"
+	"github.com/rotationalio/honu/engines/leveldb"
+	"github.com/rotationalio/honu/iterator"
+	pb "github.com/rotationalio/honu/object"
+	"github.com/rotationalio/honu/options"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
+)
+
+// Returns a constant list of namespace strings.
+// TODO: Share with honu_test.go
+func getNamespaces() []string {
+	return []string{
+		"",
+		"basic",
+		"namespace with spaces",
+		"namespace::with::colons",
+	}
+}
```

Comment:
Same comment in `honu_test.go` - any reason this needs to be a function? Also, I don't think this necessarily needs to be shared with `honu_test.go`, by keeping them independent we can try different tests. For example, the Honu tests might also test the badger and pebble engines so may have more namespace cases than is needed with leveldb (e.g. badger won't use :: prefixing).

Reply:
Same as the previous comment, I'll change it to a variable.